src/view/nova/themes.ts

```diff
@@ -0,0 +1,184 @@
+import {createTheme} from './lib/theme'
+
+const BLUE_HUE = 210
+const GRAYSCALE_SATURATION = 12
+
+export const palette = {
+  white: '#FFFFFF',
+
+  /**
+   * Mathematical scale of grays, from lightest to darkest, all based on the
+   * primary blue color
+   */
+  gray1: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 95%)`,
+  gray2: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 85%)`,
+  gray3: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 75%)`,
+  gray4: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 30%)`,
+  gray5: `hsl(${BLUE_HUE} ${GRAYSCALE_SATURATION}%, 20%)`,
+  gray6: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 10%)`,
+  black: `hsl(${BLUE_HUE}, ${GRAYSCALE_SATURATION}%, 5%)`,
+
+  blue: `hsl(${BLUE_HUE}, 100%, 53%)`,
```

Comment:
Just calling out that these differ from designs rn because the contrast is very poor in the design files. This gives us an even spread. Basically, from either end of the spectrum, there's:
- white, slightly darker white (1), slightly darker white (2), slightly darker white (3)
- black, slightly lighter black (6), slightly lighter black (5), slightly lighter black (4)